src/solidity.ts

```diff
@@ -359,6 +394,10 @@ interface ToString {
   toString(): string;
 }
 
+function isVariableDeclaration(node: solc.ast.ContractItem): node is solc.ast.VariableDeclaration {
```

Comment:
You also need to add the definition of `interface VariableDeclaration` in `solc.ts`, with the fields that are being used.

Reply:
Done.
Can you please verify that I did it right?